srcpkgs/itch/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'itch'
+pkgname=itch
+version=1.26.0
+revision=1
+archs="x86_64"
```

Comment:
Why is this arch restricted? One can run windows games on musl...

Reply:
In the case with musl, compile with archs="x86_64*" and patch_args="-Np0", but application does framework donwload to work, which are compiled for GNU libc. 

Which are these: 

https://broth.itch.ovh/itch/linux-amd64/25.5.1/archive/default

https://broth.itch.ovh/butler/linux-amd64/15.21.0/archive/default

Reply:
Does it actually need the framework to go anything useful?

Reply:
It's electron, you need to work, I opened an issue to have musl versions of the framework.

https://github.com/itchio/itch/issues/2683